### PR TITLE
[FIX, DOC] Small fixes to MSSimulator

### PR DESCRIPTION
**Pull request by cbielow**
[FIX] ignored MS2 model switch for fragment prediction (thx Sandro); 
[DOC] better parameter descriptions; code documentation, variable naming and cleanup

### CODE REVIEW

**Comment by hroest** [comment]
somehow the tests do not work, can you have a look?

**Comment by cbielow** [comment]
tests seem to be passing now

**Comment by cbielow** [comment]
merge?

File: src/openms/source/ANALYSIS/TARGETED/OfflinePrecursorIonSelection.cpp [hunk 1]
```diff
@@ -48,13 +48,13 @@ namespace OpenMS
     defaults_.setValue("ms2_spectra_per_rt_bin", 5, "Number of allowed MS/MS spectra in a retention time bin.");
     defaults_.setMinInt("ms2_spectra_per_rt_bin", 1);
 
-    defaults_.setValue("min_peak_distance", 3., "The minimal distance (in Da) of two peaks in one spectrum so that they can be selected.");
-    defaults_.setMinFloat("min_peak_distance", 0.);
+    defaults_.setValue("min_mz_peak_distance", 2.0, "The minimal distance (in Th) between two peaks for concurrent selection for fragmentation. Also used to define the m/z width of an exclusion window (distance +/- from m/z of precursor). If you set this lower than the isotopic envelope of a peptide, you might get multiple fragment spectra pointing to the same precursor.");
```
Comment (timosachsenberg) [inline src/openms/source/ANALYSIS/TARGETED/OfflinePrecursorIonSelection.cpp]: chance from 3. to 2.0 intended?

Comment (cbielow) [inline src/openms/source/ANALYSIS/TARGETED/OfflinePrecursorIonSelection.cpp]: yep, hence the adapter class tests

**Comment by timosachsenberg** [comment]
that's what I thought :)

**Pull request merged**